app/models/manageiq/providers/microsoft/infra_manager.rb

```diff
@@ -24,8 +24,7 @@ def self.raw_connect(connect_params)
     require 'winrm'
 
     connect_params[:operation_timeout] = 1800
-    winrm = WinRM::Connection.new(connect_params)
-    winrm
+    WinRM::Connection.new(connect_params)
   end
```

Comment:
Yes...I see, so this is a layer deeper.  Even so, that block paradigm for open/ensure/close is very useful as it DRYs up the pattern and futureproofs you from this problem in new locations because all callers would go through that interface